user/src/components/Layout/Header/Header.jsx

```diff
@@ -58,15 +101,31 @@ const Header = () => {
                             </div>
                             <div className='header__info col-auto'>
                                 <div className='account'>
-                                    <i className="far fa-user"></i>
-                                    <ul className='account__option shadow'>
-                                        <li className='account__item'>
-                                            Login
-                                        </li>
-                                        <li className='account__item'>
-                                            Register
-                                        </li>
-                                    </ul>
+                                    {user ?
+                                        <>
+                                            <span className='text__login'>Hi {user.userName}</span>
+                                            <ul className='account__option shadow'>
+                                                <li className='account__item'>
+                                                    Profile
+                                                </li>
+                                                <li className='account__item' onClick={logOutOnClick}>
+                                                    Logout
+                                                </li>
+                                            </ul>
+                                        </>
+                                        :
+                                        <>
+                                            <i className="far fa-user"></i>
+                                            <ul className='account__option shadow'>
+                                                <li className='account__item' onClick={loginOnClick}>
```

Comment:
```suggestion
                                                <li className='account__item' onClick={() =>
        setLogin(true)}>
```
Nếu xử lý chỉ có 1 dòng thì thôi set thẳng cho nó luôn nhé